releases/release-4.0.13.md

```diff
@@ -0,0 +1,136 @@
+---
+title: TiDB 4.0.13 Release Notes
+---
+
+# TiDB 4.0.13 Release Notes
+
+发版日期：2021 年 5 月 28 日
+
+TiDB 版本：4.0.13
+
+## 新功能
+
++ TiDB
+
+    - 支持将列属性 `AUTO_INCREMENT` 变更为 `AUTO_RANDOM` [#24608](https://github.com/pingcap/tidb/pull/24608)
+    - 引入 `infoschema.client_errors_summary` 表，用以追踪返回给客户端的错误 [#23267](https://github.com/pingcap/tidb/pull/23267)
+
+## 提升改进
+
++ TiDB
+
+    - 当内存中的统计信息缓存是最新的时，避免后台作业频繁读取 `mysql.stats_histograms` 表造成高 CPU 使用率 [#24352](https://github.com/pingcap/tidb/pull/24352)
+
++ TiKV
+
+    - 提高 `store used size` 计算过程的准确性 [9904](https://github.com/tikv/tikv/pull/9904)
+    - 在 `EpochNotMatch` 消息中返回更多的 Region 以降低 Region miss 的发生 [9731](https://github.com/tikv/tikv/pull/9731)
+    - 加快释放长期运行集群中堆积的内存 [10035](https://github.com/tikv/tikv/pull/10035)
+
++ PD
+
+    - 优化 TSO 处理时间的统计指标，帮助用户判断 PD 侧的 TSO 处理时间是否过长 [#3524](https://github.com/pingcap/pd/pull/3524)
+    - 更新 Dashboard 的版本至 v2021.03.12.1 [#3469](https://github.com/pingcap/pd/pull/3469)
+
++ TiFlash
+
+    - 自动清除过期历史数据以释放磁盘空间
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - 支持备份恢复系统库 `mysql` 下的用户表 [#1077](https://github.com/pingcap/br/pull/1077)
+        - 更新 `checkVersion` 以检查集群版本和备份数据版本 [#1090](https://github.com/pingcap/br/pull/1090)
+        - 容忍在备份期间集群中出现少数 TiKV 节点宕机 [#1062](https://github.com/pingcap/br/pull/1062)
+
+    + TiCDC
+
+        - 为内部处理单元增加流程控制，避免出现内存溢出问题 [#1751](https://github.com/pingcap/ticdc/pull/1751)
+        - 增加 Unified Sorter 清理陈旧临时文件的功能，禁止多个 `cdc` 服务共享 `sort-dir` 目录 [#1741](https://github.com/pingcap/ticdc/pull/1741)
+        - 给 Failpoint 增加 HTTP 接口调用 [#1732](https://github.com/pingcap/ticdc/pull/1732)
+
+## Bug 修复
+
```

Comment:
+ Tools
     - Lightning
         - Fix the bug that the lightning's tidb backend cannot load any data when autocommit is disabled.